README.md

```diff
@@ -4,23 +4,28 @@
 
 The project uses a virtual environment to isolate package dependencies. To run, you must ensure you have environment variables for 'KEY' and 'TOKEN' which are the key and token respectively for querying the flask API. To create the virtual environment and install required packages, run the following from a bash shell terminal:
 
-### On macOS and Linux
+### Running the Application
+
+To run this application, you must have poetry installed.
+
+After that, simply call
 ```bash
-$ source setup.sh
-```
-### On Windows (Using PowerShell)
-```powershell
-$ .\setup.ps1
+poetry install
 ```
-### On Windows (Using Git Bash)
+to install all dependencies. Then you can call
 ```bash
-$ source setup.sh --windows
+<<<<<<< HEAD
+flask run
+=======
+poetry run flask run
+>>>>>>> 2c9172557250c62664a64dd2125dbe94aa2d1d4c
```

Comment:
Looks like you had a merge conflict. You should tidy this up

Reply:
Done, ty